style.css

```diff
@@ -47,14 +47,77 @@ h1{
     padding: 10px;
 }
 
+.bio{
+    margin-top: 30px;
+    margin-bottom:15px ;
+    color: #5386E4;
+    font:normal normal normal 30px/1.4em spinnaker,sans-serif;
+    font-size: 40px;
+    text-align: center;
+    letter-spacing: 0.2em;
+}
+.str{
+    width: 100%;
+    position: relative;
+    top: 0;
+    bottom: 0;
+   }
 
-
-
-
-
-
-
-
-
-
-
+ .bio::after{
+    content:'';
+    background: #5386E4;
+    display: block;
+    height: 1px;
+    width: 900px;
+    margin: 40px auto 5px;
+}
+img{
```

Comment:
recommend using class Name not a tag type, 
by using the img tag all image in the site will take this CSS